Dockerfile

```diff
@@ -79,6 +79,8 @@ ENV VIRTUALENV_READ_ONLY_APP_DATA=1
 ARG GO=1.15.4
 ARG GO_SHA256=eb61005f0b932c93b424a3a4eaa67d72196c79129d9a3ea8578047683e2c80d5
 ENV PATH=/opt/go/bin:$PATH
+ENV GOCACHE=/tmp
```

Comment:
this can be combined with the previous `ENV` line so it's only one layer instead of two